neural_compressor/strategy/basic.py

```diff
@@ -312,6 +313,20 @@ def next_tune_cfg(self):
             stage1_max = 1e9  # TODO set a more appropriate value
             if not self.cur_best_tuning_cfg:
                 self.cur_best_tuning_cfg = deepcopy(initial_op_tuning_cfg)
+
+            # try to tune a WeightOnlyQuant algorithm
```

Comment:
Please update the trigger conditions: 
1) Currently, it's only available for the ORT backend with  `approach` is `weight_only`. 
2) It will be triggered when a) `quant_level` is `auto` or b) `quant_level` is `1` && `strategy` is `basic`.

Please also add the usage to the PR description.

Reply:
One more condition here:
3. `algorithm` is not set in op_type_dict

If user sets `algorithm`, the woq tuning will not be triggered 
```python
conf = PostTrainingQuantConfig(approach="weight_only", op_type_dict={'.*': {'weight': {'algorithm': ['RTN']}}})
```
@yiliu30 correct me if I am not right.

Reply:
> One more condition here: 3. `algorithm` is not set in op_type_dict
> 
> If user sets `algorithm`, the woq tuning will not be triggered
> 
> ```python
> conf = PostTrainingQuantConfig(approach="weight_only", op_type_dict={'.*': {'weight': {'algorithm': ['RTN']}}})
> ```
> 
> @yiliu30 correct me if I am not right.

Yes, you are right.